openrtb/openrtbtest/testutil.go

```diff
@@ -103,6 +103,10 @@ func getNumOfJsonFields(modelType reflect.Type) (result int) {
 	n := modelType.NumField()
 
 	for i := 0; i < n; i++ {
+		if modelType.Field(i).Type.Kind() == reflect.Struct {
```

Comment:
but - what does that even mean : 
[example of nonsense](https://play.golang.org/p/qAF6BoHVab)